docs/roadmap.md

```diff
@@ -1,7 +1,9 @@
-<link href="{{ site.github.url }}/tables.css" rel="stylesheet">
+<link href="{{ site.github.url }}/tables.css" rel="stylesheet" />
 
 # Version history
 
+- 0.1.5 Bug fix for a rare race condition with time-out on blocking molecules. Error reporting via data structures rather than flat strings. New `B#checkTimeout` API to make a clean distinction between replies that need to check the timeout status and replies that don't. Documentation was improved. Code cleanups resulted in 98% test coverage.
```

Comment:
This is major PR to me. You could mention SHA fix for 2.11 and 2.12 differences; I don't think it's  a "minor" contribution.